Sample Data/Custom/DSTIMAccess_CL.json

```diff
@@ -0,0 +1,62 @@
+[
+  {
+    "TenantId": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+    "SourceSystem": "RestAPI",
+    "MG": "",
+    "ManagementGroupName": "",
+    "TimeGenerated": "2021-11-18T21:58:38.0289591Z",
+    "Computer": "",
+    "RawData": "",
+    "TimeDiscovered": "2021-06-18T21:58:38.0289591Z",
+    "Location": "eastus",
+    "OperationName": "GetBlob",
+    "StatusCode": "200",
+    "Uri": "https://aipclptest.blob.core.windows.net/test2/es-ssn (1).docx",
+    "CallerIPAddress": "10.1.0.38",
+    "UserAgentHeader": "Azure-Storage/9.3.0 (.NET CLR 4.0.30319.42000; Win32NT 6.2.9200.0)",
+    "ResourceSubscriptionId": "bbbabe37-eda3-48ea-98ed-f0a70c31a45b",
+    "ResourceGroup": "BreachManagement-dev",
+    "StorageAccountName": "breachmanagement-dev",
+    "ResponseBodySize": "26070",
+    "RequesterUpn": "MeganB@seccxp.ninja",
```

Comment:
Email should be sanitized@sanitized.com.

Reply:
@NikTripathi fixed.